src/main/java/seedu/duke/model/exception/IllegalFormatException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.duke.model.exception;
+
+public class IllegalFormatException extends Exception {
+    public IllegalFormatException(String message) {
+        super(message);
+    }
+}
```

Comment:
it does not belong to model package
also there is one in the parser class, but I'm not sure where did this exception is used. please check and correct accordingly

Reply:
it is only used in parser. I'll delete it elsewhere